phing/tasks/tests.xml

```diff
@@ -83,21 +83,45 @@
   <target name="tests:phpunit" description="Executes PHPUnit tests against custom code.">
     <delete dir="${reports.localDir}/phpunit" quiet="true"/>
     <mkdir dir="${reports.localDir}/phpunit"/>
+    <if>
+      <and>
+        <isset property="phpunit.local" />
+        <not><isset property="env.TRAVIS" /></not>
+      </and>
+      <then>
+        <property name="phpunit.dirs" value="${phpunit.local}" />
+      </then>
+
+      <elseif>
+        <and>
+          <isset property="phpunit.deploy" />
+          <isset property="env.TRAVIS" />
+        </and>
+        <then>
+          <property name="phpunit.dirs" value="${phpunit.deploy}" />
+        </then>
+      </elseif>
+
+      <else>
+        <property name="phpunit.dirs" value="${repo.root}/tests/phpunit,${blt.root}/tests/phpunit/BltProject" />
+      </else>
+    </if>
+    <foreach list="${phpunit.dirs}" param="dir" target="tests:phpunit:dir"/>
+  </target>
 
+  <target name="tests:phpunit:dir" description="Run phpunit against given directories." hidden="true">
     <phpunit printsummary="true" failureproperty="phpunit.failures" errorproperty="phpunit.errors">
       <formatter type="plain" usefile="false"/>
       <formatter type="xml" todir="${reports.localDir}/phpunit"/>
       <batchtest>
-        <fileset dir="${repo.root}/tests/phpunit">
+        <fileset dir="${dir}">
```

Comment:
Rather than specifying dirs for local and deploy as a means of grouping tests, why not use PHPUnit's built-in ability to group tests with `@` tags? We can make use of the [phpunit task's](https://www.phing.info/docs/stable/hlhtml/index.html#PHPUnitTask) group and excludeGroup params to accomplish this.

Following the established naming conventions for BLT targets, we can then have `ci:tests:phpunit` and `local:tests:phpunit` targets that simply pass params value for group and excludeGroup to `tests:phpunit`. 

Users can also then directly call `tests:phpunit -Dphpunit.groups='' -Dphpunit.excludeGroups=''` to set custom values.

Even with this approach, it would still be great to turn phpunit.dirs into a variable and set a default value in build.yml. E.g.,

    phpunit:
      dirs: 
        - ${repo.root}/tests/phpunit

This would allow users to easily override the directories by setting a custom value in project.yml.